app/src/main/java/de/westnordost/streetcomplete/util/ktx/LocalDate.kt

```diff
@@ -17,3 +36,59 @@ fun isApril1st(): Boolean {
     val now = LocalDate.now()
     return now.dayOfMonth == 1 && now.month == Month.APRIL
 }
+
+inline fun Instant_Companion.ofEpochMilli(epochMilliseconds: Long): Instant =
+    Instant.fromEpochMilliseconds(epochMilliseconds)
+
+inline fun LocalDate_Companion.now(): LocalDate = Clock.System.now().toLocalDate()
+
+inline fun LocalDate_Companion.of(year: Int, monthNumber: Int, dayOfMonth: Int): LocalDate =
+    LocalDate(year, monthNumber, dayOfMonth)
+
+fun LocalDateTime_Companion.now(): LocalDateTime =
+    Clock.System.now().toLocalDateTime(ZoneId.systemDefault())
+
+inline fun LocalDateTime_Companion.of(date: LocalDate, time: LocalTime): LocalDateTime =
+    LocalDateTime(date, time)
+
+inline fun LocalDateTime_Companion.ofInstant(instant: Instant, zone: ZoneId): LocalDateTime =
+    instant.atZone(zone)
+
+inline fun LocalTime_Companion.ofSecondOfDay(secondOfDay: Long): LocalTime =
+    LocalTime.fromSecondOfDay(secondOfDay.toInt())
+
+inline fun TimeZone_Companion.systemDefault(): ZoneId = ZoneId.currentSystemDefault()
+
+inline fun Instant.atZone(timeZone: ZoneId): LocalDateTime = this.toLocalDateTime(timeZone)
+
+inline fun Instant.toEpochMilli(): Long = this.toEpochMilliseconds()
```

Comment:
As I wrote in other comment, here and for other functions, it is OK IMO to use the kotlinx-datetime API and not try to mirror the java.time API as long as it is not more complicated. Some additional letters don't matter :-P

Reply:
https://github.com/streetcomplete/StreetComplete/pull/4471/commits/05906c64861f6b10b777604588ebc6bf99c1d478